pkg/ottl/ottlfuncs/README.md

```diff
@@ -1033,6 +1034,32 @@ Examples:
 
 **Note:** According to the National Institute of Standards and Technology (NIST), MD5 is no longer a recommended hash function. It should be avoided except when required for compatibility. New uses should prefer a SHA-2 family function (e.g. SHA-256, SHA-512) whenever possible.
 
+### MarshalKeyValue
+
+`MarshalKeyValue(target, Optional[delimiter], Optional[pair_delimiter])`
+
+The `MarshalKeyValue` Converter takes a `pcommon.Map` and converts it to a `string` of key value pairs.
+
+- `target` is a Getter that returns a `pcommon.Map`. 
+- `delimiter` is an optional string that is used to join keys and values, the default is `=`. 
+- `pair_delimiter` is an optional string that is used to join key value pairs, the default is a single space (` `).
```

Comment:
Is ` ` the proper delimiter for windows event logs?

Reply:
I know the linked issue mentioned WEL but as I understand it, the WEL XML would have already been parsed into a generic map and this function just converts that generic map into a string of a different format. WEL format isn't trying to be preserved, so the ` ` isn't really related.